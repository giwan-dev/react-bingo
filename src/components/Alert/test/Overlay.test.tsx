import { shallow } from 'enzyme';
import React from 'react';
import AlertOverlay from '../Overlay';

describe('AlertOverlay component', () => {
  it('should render withount crashing', () => {
    const wrapper = shallow((
      <AlertOverlay />
    ));

    expect(wrapper).toMatchSnapshot();
  });
});
