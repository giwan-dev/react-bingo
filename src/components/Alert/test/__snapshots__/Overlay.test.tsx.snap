// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertOverlay component should render withount crashing 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background-color: rgba(0, 0, 0, 0.4);
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
